catalyst/utils/config.py

```diff
@@ -74,6 +86,8 @@ def parse_config_args(*, config, args, unknown_args):
             else:
                 arg_value = eval("%s(%s)" % (value_type, value_content))
             args.__setattr__(arg_name, arg_value)
+            set_arg_to_config(config, arg_name, arg_value)
```

Comment:
why don't you just move
```
for key, value in args._get_kwargs():
    set_arg_to_config(config, key, value)
```
to the end of function, before the `return config, args`?

Reply:
updated